cloudify_cli/tests/commands/test_sites.py

```diff
@@ -0,0 +1,96 @@
+########
+# Copyright (c) 2013-2019 Cloudify Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+
+from .test_base import CliCommandTest
+from cloudify_cli.exceptions import CloudifyValidationError, CloudifyCliError
+
+
+class SitesTest(CliCommandTest):
```

Comment:
Positive tests are missing

Reply:
There is no good way to do positive cli test because there is no manager, so I can mock it but I don't think there is a value.
It is just the cli tests, I have positive tests of the rest api.

Reply:
yes, at least a basic test which will not get an exception of some sort